src/assets/css/sass/admin/_address.scss

```diff
@@ -6,7 +6,7 @@
     }
 
     button.add {
-      background: $blue;
+      background: #4B5CF0;
```

Comment:
Why don't we store it in a variable to be reused?

Reply:
Acutally, I've seen that we changed the `$blue` variable, so we shouldn't need to specify it here, should we?

Reply:
I'l reverted the changes and use `$blue` with the value modified